### DISCUSSION TITLE
Error with Making Jsonnet

### DISCUSSION THREAD

**Discussion by oheckmann** [carlosedp/cluster-monitoring#165] [Q&A]
Good Afternoon, 
I think that this is a great out-of-the-box solution for a problem that I have. I am working on following the K3s Quickstart Guide.

However, when I run `sudo make`, I get the following errors

`
Installing jsonnet
go: found github.com/google/go-jsonnet/cmd/jsonnet in github.com/google/go-jsonnet v0.19.1
go: github.com/google/go-jsonnet latest => v0.19.1
# golang.org/x/sys/unix
/root/go/pkg/mod/golang.org/x/sys@v0.1.0/unix/syscall.go:83:16: undefined: unsafe.Slice
/root/go/pkg/mod/golang.org/x/sys@v0.1.0/unix/syscall_linux.go:2255:9: undefined: unsafe.Slice
/root/go/pkg/mod/golang.org/x/sys@v0.1.0/unix/syscall_unix.go:118:7: undefined: unsafe.Slice
/root/go/pkg/mod/golang.org/x/sys@v0.1.0/unix/sysvshm_unix.go:33:7: undefined: unsafe.Slice
note: module requires Go 1.17
make: *** [Makefile:56: /root/go/bin/jsonnet] Error 2
`

Is this a known issue? If so, how would I best correct this?

**Comment by carlosedp** [comment]
Might be a problem with go-jsonnet dependencies... better to try it separately and if the case, open an issue there.
